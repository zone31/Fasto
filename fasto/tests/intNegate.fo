fun int test(int x) = 
    let w = write (x)  in
    0 

fun int main() =
    let w =  test(3/2) in
    let w =  test(5/3) in
    let w =  test(10/9) in
    let w =  test(10/11) in
    let w =  test(~10/2) in
    let w =  test(~10/~2) in
    let w =  test(10/~2) in
    
    0
    
