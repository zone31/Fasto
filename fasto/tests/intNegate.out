1 1 1 0 -5 5 -5 -1 15 -8